FormlyMapper/FormlyField/ChoiceField.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class ChoiceField extends FormlyField
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFieldType()
+    {
+        return 'select';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildFieldTypeConfiguration()
+    {
+        $type = $this->getFieldType();
+
+        $typeOptions = $type;
+        $expanded = false;
+        $multiple = false;
+
+        if (isset($this->fieldConfiguration['options']['expanded'])) {
+            $expanded = $this->fieldConfiguration['options']['expanded'];
+        }
+
+        if (isset($this->fieldConfiguration['options']['multiple'])) {
+            $multiple = $this->fieldConfiguration['options']['multiple'];
+        }
+
+        if ($expanded) {
+            $type = 'radio';
+            if ($multiple) {
```

Comment:
Need a new line here to separate the condition block.https://github.com/LinioIT/linio-standards/blob/master/standards/php-coding-standard.md#newlines